src/Tizen.Wearable.CircularUI.Forms.Renderer/Map/MapService.cs

```diff
@@ -0,0 +1,142 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using Tizen.Wearable.CircularUI.Forms.Renderer.Map.Geocoding;
+using Tizen.Wearable.CircularUI.Forms.Renderer.Map.Utils;
+using Xamarin.Forms.Maps;
+
+namespace Tizen.Wearable.CircularUI.Forms.Renderer.Map
+{
+    public class MapService
```

Comment:
GoogleMapService?

Reply:
MapService is only for Geocoding API. I deleted this class